install/helm/agones/defaultfeaturegates.yaml

```diff
@@ -19,11 +19,11 @@ CustomFasSyncInterval: true
 SDKGracefulTermination: true
 StateAllocationFilter: true
 PodHostname: true
+ResetMetricsOnDelete: true
```

Comment:
can you fix the ordering here? It should be alphabetical within the stage. Looks like `PodHostname` also violates right now.

Reply:
rearranged it